src/api/controllers/articleController.js

```diff
@@ -53,6 +52,110 @@ export default class ArticleController {
     }
   }
 
+  /**
+   * publish a drafted article
+   * @static
+   * @param {Object} req the request
+   * @param {Object} res the response to be sent
+   * @memberof ArticleController
+   * @returns {Object} res
+   */
+  static async publish(req, res) {
+    const { slug } = req.params;
+    const article = await Article.update({
+      status: 'published'
+    },
+    {
+      where: {
+        slug,
+        title: {
+          [Sequelize.Op.ne]: null,
+        },
+        body: {
+          [Sequelize.Op.ne]: null,
+        },
+        description: {
+          [Sequelize.Op.ne]: null,
+        },
+        category: {
+          [Sequelize.Op.ne]: null,
+        },
+        coverImage: {
+          [Sequelize.Op.ne]: null,
+        },
+      }
+    });
+    const published = 1;
```

Comment:
@alainburindi what do you think about `publishedState` ?

Reply:
s just a number which represents if the article has been published or not